tests/test_imports.py

```diff
@@ -0,0 +1,45 @@
+import platform
+import sys
+
+import pytest
+
+
+def test___all__(pytester: pytest.Pytester) -> None:
+    """See https://github.com/aio-libs/aiohttp/issues/6197"""
```

Comment:
It'd be useful to see an actual docstring here, not a reference to something external.

Reply:
I've not changed this test, the diff is just confused due to the file rename.

Reply:
So maybe do this in a separate PR, then?